README.md

```diff
@@ -19,3 +19,26 @@ Follow the instructions in [the Local CHT Sync Setup documentation](https://docs
 # set environment variables, install dbt dependencies, seed data, run dbt, run test
 ./run_dbt_tests.sh
 ```
+
+## Release Process
+This repo has an automated release process where each feature/bug fix will be released immediately after it is merged to `main`:
+
+1. Update QA with the work to be done to ensure they're informed and can guide development.
```

Comment:
Can we link the [existing documentation](https://docs.communityhealthtoolkit.org/contribute/code/workflow/) about the CHT code workflow to ensure consistency across all the repos? These steps shouldn't be different from the ones stated in the doc.